src/main/java/core/utils/TrabajopolisUtils.java

```diff
@@ -54,6 +53,15 @@ public void click(final By locator) {
         driver.findElement(locator).click();
     }
 
+    /**
+     * This method makes click action.
+     */
+    public void clickMulti() {
```

Comment:
Is not understanding the name of method, please change the name.

Reply:
done